packages/cli/bin/n8n

```diff
@@ -47,6 +47,9 @@ if (process.env.NODEJS_PREFER_IPV4 === 'true') {
 	require('dns').setDefaultResultOrder('ipv4first');
 }
 
+// set Socket `autoSelectFamily` defaults to pre v20 behavior
```

Comment:
Maybe elaborate a bit more on _why_ we are setting this

Reply:
I'm still trying to get a better understanding of this issue, and not sure what else to add here that would make sense.
Any suggestions? if not, I say that we leave it as it is, and let people find this PR via git-blame for now.
This is hopefully a temporary solution, until the algorithm is improved in node, or we find a way to tweak more of its settings.

Reply:
I was thinking something along the lines:

```suggestion
// Node.js 20 enabled a Happy Eyeballs algorithm which enables support
// for both IPv6 and IPv4 at the same time, favoring IPv6 when possible.
// However there are some issues in the algorithm implementation that is causing
// issues to our users with services like Telegram or Airtable. This restores the
// behavior to pre v20
// More details: https://github.com/nodejs/node/issues/48145
```